flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/DefaultTimeServiceProvider.java

```diff
@@ -99,7 +100,7 @@ public void run() {
 					target.trigger(timestamp);
 				} catch (Throwable t) {
 					TimerException asyncException = new TimerException(t);
```

Comment:
Although, now that I think about it, it is good to know that it came from a timer callback. What do you think?